src/main/java/org/gatorgradle/display/CheckResult.java

```diff
@@ -31,13 +31,16 @@ public MalformedJsonException(String reason, String json) {
   public static final String PASS_SYMBOL_RAW = "\u2714"; // heavy check (✔)
   public static final String FAIL_SYMBOL_RAW = "\u2718"; // heavy cross (✘)
   public static final String FIX_SYMBOL_RAW = "\u2794"; // right arrow (➔)
+  public static final String REACH_SYMBOL_RAW = "\u25CF"; // circle (●)
 
   public static final String PASS_SYMBOL =
       StringUtil.color(StringUtil.GOOD, PASS_SYMBOL_RAW);
   public static final String FAIL_SYMBOL =
       StringUtil.color(StringUtil.BAD, FAIL_SYMBOL_RAW);
   public static final String FIX_SYMBOL =
       StringUtil.color("\u001B[1;33m", FIX_SYMBOL_RAW);
+  public static final String REACH_SYMBOL =
+      StringUtil.color("\u001B[36m", REACH_SYMBOL_RAW);
```

Comment:
It might be a good idea to make names and constants for these colors in StringUtil, just like the `GOOD` and `BAD` colors -- something like `PROBLEM` and `GOAL` or so on.

Reply:
Also, it would be great if you could summarize the intent of your changes in the PR body.